sql/hive/src/main/scala/org/apache/spark/sql/hive/client/ClientWrapper.scala

```diff
@@ -166,7 +168,11 @@ private[hive] class ClientWrapper(
           " specified in spark.yarn.keytab does not exist")
       } else {
         logInfo("Attempting to login to Kerberos" +
-          s" using principal: ${principalName} and keytab: ${keytabFileName}")
+        s" using principal: ${principalName} and keytab: ${keytabFileName}")
+        val hadoopConfiguration = new Configuration()
+        hadoopConfiguration.set(CommonConfigurationKeysPublic.HADOOP_SECURITY_AUTHENTICATION,
```

Comment:
Hi!
I think it is required because of another context of execution in this part of code. Here is a context of the classloader, which cause an empty environment, with default value of authentication, which is "simple". In case of "simple" a keytab is not used and credentials cannot be granted.